code/go/0chain.net/validatorcore/storage/handler_integration_tests.go

```diff
@@ -0,0 +1,13 @@
+//go:build integration_tests
```

Comment:
looks like this file is the same as handler_main.gp

Reply:
It is. This is part of validator conductor tests development. We'll need to override the endpoint handler `/v1/storage/challenge/new`.